qiita_ware/commands.py

```diff
@@ -70,22 +71,41 @@ def submit_EBI(preprocessed_data_id, action, send, fastq_dir_fp=None):
     ebi_submission.write_xml_file(ebi_submission.generate_submission_xml(),
                                   ebi_submission.submission_xml_fp)
 
-    # other steps
     if send:
-        ebi_submission.send_sequences()
-        study_accession, submission_accession = ebi_submission.send_xml()
+        # step 4: sending sequences
+
+        # Set the ASCP password to the one in the Qiita config, but remember
+        # the old pass so that we can politely reset it
+        old_ascp_pass = environ.get('ASPERA_SCP_PASS', '')
```

Comment:
Oh! Sorry, does `generate_send_sequences_cmd` issue the system call? Didn't realize that. If so, then I do still agree that it would be more explicit to move the environment manipulation to as close to the system call as possible.